### PR TITLE
Merge dev the master (helm chart fix)

### PR DESCRIPTION
**Pull request by 27149chen**
Only with some helm chart alignments with ICP certification.

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -110,7 +110,7 @@ import:
   subpackages:
   - lib
 - package: github.com/IBM/ubiquity
-  version: e1ee55cbe8eb3cc972b6a33e00b792c51923b254
+  version: 36cac8b058f911c94b3730b2d4f56050db388ce1
```
Comment (shay-berman) [inline glide.yaml]: Did you change something in ubiquity repo?  if not why a new hash here?

File: helm_chart/ibm-storage-enabler-for-containers-dev/cv-tests/test-01/values.yaml [hunk 2]
```diff
@@ -18,7 +18,7 @@ backend: spectrumConnect
 spectrumConnect:
   connectionInfo:
     # IP\FQDN and port of Spectrum Connect server.
-    fqdn: 9.37.37.46
+    fqdn: spectrum-connect.rtp.raleigh.ibm.com
```
Comment (shay-berman) [inline helm_chart/ibm-storage-enabler-for-containers-dev/cv-tests/test-01/values.yaml]: r u sure its mandatory to state this host here?

File: glide.yaml (hunk 1 above)
Comment (27149chen) [reply glide.yaml]: one is on dev, one is on master, never the same.

File: helm_chart/ibm-storage-enabler-for-containers-dev/cv-tests/test-01/values.yaml (hunk 2 above)
Comment (27149chen) [reply helm_chart/ibm-storage-enabler-for-containers-dev/cv-tests/test-01/values.yaml]: yes

**Pull request merged**